tests/integration/tier0/non-destructive/kernel/test_custom_kernel.py

```diff
@@ -21,23 +21,28 @@
         "grub_substring": "Oracle Linux Server (4.18.0-348.el8.x86_64) 8.5",
     },
     "oracle-7": {
-        "original_kernel": f"{ORIGINAL_KERNEL}",
+        "original_kernel": ORIGINAL_KERNEL,
         "custom_kernel": "http://mirror.centos.org/centos/7/os/x86_64/Packages/kernel-3.10.0-1160.el7.x86_64.rpm",
         "grub_substring": "Oracle Linux Server 7.9, with Linux 3.10.0-1160.el7.x86_64",
     },
     # Install CentOS 8.5 kernel
     "oracle-8-latest": {
-        "original_kernel": f"{ORIGINAL_KERNEL}",
+        "original_kernel": ORIGINAL_KERNEL,
         "custom_kernel": "https://vault.centos.org/centos/8.5.2111/BaseOS/x86_64/os/Packages/kernel-core-4.18.0-348.7.1.el8_5.x86_64.rpm",
         "grub_substring": "CentOS Linux (4.18.0-348.7.1.el8_5.x86_64) 8",
     },
     "alma-8": {
-        "original_kernel": f"{ORIGINAL_KERNEL}",
+        "original_kernel": ORIGINAL_KERNEL,
         "custom_kernel": "https://yum.oracle.com/repo/OracleLinux/OL8/5/baseos/base/x86_64/getPackage/kernel-core-4.18.0-348.el8.x86_64.rpm",
         "grub_substring": "Oracle Linux Server (4.18.0-348.el8.x86_64) 8.5",
     },
     "rocky-8": {
-        "original_kernel": f"{ORIGINAL_KERNEL}",
+        "original_kernel": ORIGINAL_KERNEL,
+        "custom_kernel": "https://yum.oracle.com/repo/OracleLinux/OL8/5/baseos/base/x86_64/getPackage/kernel-core-4.18.0-348.el8.x86_64.rpm",
```

Comment:
Oracle kernel to rule them all! :D

Reply:
<img src="https://media4.giphy.com/media/ie8I61aEWnJCM/giphy.gif"/>